lib/Sema/CSDiag.cpp

```diff
@@ -1022,6 +1023,7 @@ CalleeCandidateInfo::evaluateCloseness(Type candArgListType,
   // type variables for all generics and solve it with the given argument types.
   Type singleArchetype = nullptr;
   Type matchingArgType = nullptr;
+  unsigned nonSubstitutableArgs = 0;
```

Comment:
Please a comment on this explaining what it is about, since it isn't obvious from the name.